modules/internal-certificate-authority-rotation-for-rhacs.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * configuration/reissue-internal-certificates.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="internal-certificate-authority-rotation-for-rhacs_{context}"]
+= Internal certificate authority rotation for RHACS
+
+The following points explain why certificate authority (CA) rotation is important in {rh-rhacs-first}:
+
+* {product-title-short} includes an internal CA with a 5-year validity. Central and secured cluster service certificates depend on this CA. If the CA expires, {product-title-short} stops working.
+* Previously, you had to delete the `central-tls` secret and manually re-register all the secured clusters. With this update, the Operator manages CA rotation. Central simultaneously trusts two CAs, enabling overlap and a smoother migration.
```

Comment:
"with this update" reads like release notes, but this will be part of the documentation of future versions too, right?